src/arcade/patch/agent/module/PatchModule.java

```diff
@@ -17,7 +17,7 @@ public abstract class PatchModule implements Module {
     final PatchLocation location;
 
     /** Tick the {@code Module} was started. */
-    double start;
+    int start;
```

Comment:
Move this variable into `PatchModuleProliferation`, since it is only used by `PatchModuleProliferation`